README.md

```diff
@@ -31,6 +31,12 @@ Step 5. ```vagrant up```
 
 From here, you should be able to browse to http://localhost:8080/. The website is served with NGINX and PHP 7.
 
+### Getting started with Docker
+If you're a Docker user rather than Vagrant, there is a `docker-compose set up provided`.
```

Comment:
I put my quotes in the wrong place. Docker-compose is the orchestration tool provided, if people are using Docker locally the chances are they'll have compose installed as well. I have put my `` in the wrong place so I'll move those